packages/web-react/src/components/FileUploader/useFileUploaderAttachment.ts

```diff
@@ -1,20 +1,44 @@
 import { RefObject, useEffect } from 'react';
-import { getAttachmentInput } from './utils';
+import { getAttachmentInput, getAttachmentMetaInput } from './utils';
+import { FileQueueValueMetaType } from '../../types/fileUploader';
 
 export interface UseFileUploaderAttachmentProps {
   attachmentRef: RefObject<HTMLLIElement>;
   file: File;
+  meta?: FileQueueValueMetaType;
   name: string;
   onError?: (error: string) => void;
 }
 
-export const useFileUploaderAttachment = ({ attachmentRef, file, name, onError }: UseFileUploaderAttachmentProps) => {
+export const useFileUploaderAttachment = ({
+  attachmentRef,
+  file,
+  meta,
+  name,
+  onError,
+}: UseFileUploaderAttachmentProps) => {
   const createAttachmentInput = () => {
     const attachmentInputElement = getAttachmentInput(file, name, onError) as Node;
 
     attachmentRef.current?.appendChild(attachmentInputElement);
+
+    if (meta) {
+      const attachmentMetaInputElement = getAttachmentMetaInput(file, name, meta) as Node;
```

Comment:
It is necessary to use `as Node`? I think it would be better to return correct type/set correct return type of the function.